core/arch/arm/kernel/cache_helpers_a64.S

```diff
@@ -130,9 +128,7 @@ loop1:
 	br	x14			// jump to DC operation specific loop
 
 	.macro	dcsw_loop _op
-#if defined(CFG_CORE_BTI)
-	bti	j
-#endif
+	BTI(BTI_J)
```

Comment:
Any particular reason to use the macro `BIT_J` instead of using the instruction `bti j`?

Reply:
No, I was adding one for BTI_C for asm.S file so added for bti j too.

Reply:
I would also prefer `BTI(bti j)` here and everywhere else :)